bundles/binding/org.openhab.binding.bticino/src/main/java/org/openhab/binding/bticino/internal/BticinoBinding.java

```diff
@@ -61,12 +61,17 @@
         String host;
         // Default port is 20000 for a MH200
         int port = 20000;
+
+        // Default OpenWenNet password
+        String passwd = "00000";
```

Comment:
How does this new default password affect existing installations, where there is no password?

Reply:
When the WebServer/Gateway is configured to not use the password, this value is not even used, no matter the initial value configured. The "00000" has been chosen for convenience when using gateways like the touch panel H/L4684, which has, by default, this value and cannot be configured without password.